sync.py

```diff
@@ -186,7 +196,28 @@ def post_authentic(config, body, signature):
     return True
     return signature == expected
 
+def delete_issue_comments(issue_number):
+    """Delete all comments in the issue written by the configured user."""
+    user_name = config["username"]
+    auth = (user_name, config["password"])
+    issues_url = "https://api.github.com/repos/%s/%s/issues/" % (config["org_name"], config["repo_name"])
+    issue_comments = requests.get(urljoin(issues_url, "%s/comments" % issue_number), auth=auth).json()
+
+    # Assuming that some bug or other condition may have caused multiple
+    # comments from this bot, delete them all.
+    for comment in issue_comments:
+        if comment["user"]["login"] == user_name and "These tests are now available" in comment["body"]:
```

Comment:
It would be nice if we had something better than this string to use as a comment identifier, but I don't have a great suggestion.